ever2simple/converter.py

```diff
@@ -100,4 +108,19 @@ def _convert_json(self, notes):
         if self.simple_filename is None:
             sys.stdout.write(json.dumps(notes))
         else:
-            json.dump(notes, self.simple_filename)
+            with open(self.simple_filename, 'w') as output_file:
+                json.dump(notes, output_file)
+
+    def _convert_dir(self, notes):
+        if self.simple_filename is None:
+            sys.stdout.write(json.dumps(notes))
+        else:
+            if os.path.exists(self.simple_filename) and not os.path.isdir(self.simple_filename):
+                print '"%s" exists but is not a directory. %s' % self.simple_filename
+                sys.exit(1)
+            elif not os.path.exists(self.simple_filename):
+                os.makedirs(self.simple_filename)
+            for i, note in enumerate(notes):
+                output_file_path = os.path.join(self.simple_filename, str(i) + '.txt')
```

Comment:
I thought about that first, but I found the following issues:

* It would be needed to parse the `note['content']` to read the first line, taking the markdown out (to just have the title without the `#`)
* The title could have characters not allowed by the OS (depending on the OS), not allowing to create the file or creating a misbehaving file.
* The title could lead to security issues
  * These two things could probably be solved using [`werkzeug.utils.secure_filename`](http://werkzeug.pocoo.org/docs/0.11/utils/#werkzeug.utils.secure_filename), but that would mean adding another dependency
* As Simplenote doesn't read the file title, it seemed enough for me (and easier / faster) to have just numbered files.
* I also thought that one way to get the notes named by the titles would be to:
  * Unlink the Dropbox account
  * Remove the files in the `Simplenote` directory
  * Re-enable Drobox sync (Simplenote would then sync and name the notes using their titles)
* In case there were two notes with the same title, one would override the other. So it would be needed to add some book keeping of the titles already used, etc.
* Also, I wasn't sure you would check the PR (that you certainly did :smile: ), so I just focused on my own simple use case first.

What's your opinion? Do you think it's worthwhile the effort of parsing the title, adding more dependencies, book keeping of names, etc?